src/components/inputs/radio-input/README.md

```diff
@@ -8,46 +8,65 @@ import { RadioInput } from '@commercetools-frontend/ui-kit';
 
 #### Description
 
-A `RadioInput` is grouped within a set of disjoint items where one item should always
-be selected. As a result the named `RadioInput` export exposes a `Group` and an
-`Option`.
+A `RadioInput` represents a group of options where only one option can be selected at a time.
+The exported component contains a `RadioInput.Group` and `RadioInput.Option` components, with the `RadioInput.Group` component
+being the wrapped around the list of `RadioInput.Option`s.
+
+Most of the props should be assigned to the `RadioInput.Group` component, which internally forwards them
+to each `RadioInput.Option` component. This is mostly to avoid passing a lot of props to each `RadioInput.Option` component.
 
 #### Usage
 
 ```js
 import { RadioInput } from '@commercetools-frontend/ui-kit/switches';
 
-<RadioInput.Group onChange={...} value="foo-radio-value">
-   <RadioInput.Option value="foo-radio-value">
-      What ever option
-   </RadioInput.Option>
-   <RadioInput.Option value="bar-radio-value" isDisabled={true}>
-      Another option
-   </RadioInput.Option>
+<RadioInput.Group onChange={...} name="fruits" value="apples">
+  <RadioInput.Option value="apple">
+    <Spacings.Inline alignItems="center">
+      <div>🍎</div>
+      Apple
+    </Spacings.Inline>
+  </RadioInput.Option>
+  <RadioInput.Option value="banana">
+    <Spacings.Inline alignItems="center">
+      <div>🍌</div>
+      Banana
+    </Spacings.Inline>
+  </RadioInput.Option>
 </RadioInput.Group>
 ```
 
 #### Properties
 
 ##### RadioInput.Group
 
-| Props      | Type     | Required | Values                      | Default | Description                                                                       |
-| ---------- | -------- | :------: | --------------------------- | ------- | --------------------------------------------------------------------------------- |
-| `children` | `node`   |    ✅    | -                           | -       | The `RadioInput.Option` or another node (mixed children are allowed)              |
-| `value`    | `string` |    -     | -                           | -       | The selected value of the of a `RadioInput.Option` determining its checked status |
-| `onChange` | `func`   |    ✅    | -                           | -       | What will trigger whenever an `RadioInput.Option` is clicked                      |
-| `scale`    | `string` |    -     | ['xs', 's', 'm', 'l', 'xl'] | `m`     | Spacing between options                                                           |
```

Comment:
Technically, this is what causes the breaking change, as it's been renamed to `directionProps: { scale }`